blocks/serialization.py

```diff
@@ -136,11 +136,12 @@
 from blocks.config import config
 from blocks.filter import get_brick
 from blocks.utils import change_recursion_limit
+from blocks.bricks.base import BRICK_PATH_DELIMITER
 
 
 logger = logging.getLogger(__name__)
 
-BRICK_DELIMITER = '|'
+SERIALIZATION_BRICK_PATH_DELIMITER = '|'
```

Comment:
Well, it is used only once, so I do not think that this very long name is
an issue. On the other hand it is super clear what it is used for.

On 30 May 2016 at 16:13, dmitriy-serdyuk <notifications@github.com> wrote:

> In blocks/serialization.py
> <https://github.com/mila-udem/blocks/pull/1097#discussion_r65103495>:
>
> >
> >
> >  logger = logging.getLogger(__name__)
> >
> > -BRICK_DELIMITER = '|'
> > +SERIALIZATION_BRICK_PATH_DELIMITER = '|'
>
> Oh, that's a bit too much ;) Maybe just SERIALIZATION_PATH_DELIMITER?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/mila-udem/blocks/pull/1097/files/f6a4d09728f529d77cb00e4b81bbf86093a49143#r65103495>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAn8YmhDyZavy5SNJN7v1kZepaczOe8Hks5qG0TOgaJpZM4Ip_g7>
> .
>